demos/jans-tarp/src/options/userDetails.tsx

```diff
@@ -61,23 +62,23 @@ const UserDetails = (data) => {
             </div>
             <legend><span className="number">O</span> User Details:</legend>
             <hr />
-            {data.data.displayToken ?
+            {data.displayToken ?
```

Comment:
Please check displayToken is exist in data for not. For example - data?.displayToken.

Reply:
done